src/datasources/blockchain/blockchain-api.manager.spec.ts

```diff
@@ -32,6 +32,45 @@ describe('BlockchainApiManager', () => {
 
       expect(api).toBe(cachedApi);
     });
+
+    it('should include the INFURA_API_KEY in the RPC URI for an Infura URI with API_KEY_PATH authentication', async () => {
+      const rpcUriValue = `${faker.internet.url()}infura${faker.string.sample()}`;
+      const chain = chainBuilder()
+        .with(
+          'rpcUri',
+          rpcUriBuilder()
+            .with('value', rpcUriValue)
+            .with('authentication', RpcUriAuthentication.ApiKeyPath)
+            .build(),
+        )
+        .build();
+      configApiMock.getChain.mockResolvedValue(chain);
+
+      const api = await target.getBlockchainApi(chain.chainId);
+
+      expect(api.chain?.rpcUrls.default.http[0]).toContain(infuraApiKey);
+    });
+
+    it('should not include the INFURA_API_KEY in the RPC URI for an Infura URI without API_KEY_PATH authentication', async () => {
+      const rpcUriValue = `${faker.internet.url()}infura${faker.string.sample()}`;
```

Comment:
As above regarding the URL value.

Reply:
Changed in https://github.com/safe-global/safe-client-gateway/pull/1644/commits/234dd4de3a9efc54df8ba060a91d4815c3cb4b6e